runtime/domain.c

```diff
@@ -1470,17 +1532,18 @@ int caml_try_run_on_all_domains_with_spin_work(
   CAML_EV_BEGIN(EV_STW_LEADER);
   caml_gc_log("causing STW");
 
-  /* setup all fields for this stw_request, must have those needed
-     for domains waiting at the enter spin barrier */
+  /* set up all fields for this stw_request, must have those needed
```

Comment:
Replace the comma with a period? Not sure I understand the second half of this comment.